tools/checkcomments/testdata/testdata.go

```diff
@@ -16,13 +16,21 @@
 package testdata
 
 func testCorrect() {
-	// TODO https://github.com/FerretDB/FerretDB/issues/2733
+	// TODO https://github.com/FerretDB/FerretDB/issues/3413
 }
 
 func testCorrectForNow() {
 	// TODO no URL
 }
 
-func testIncorrect() {
-	// TODO: https://github.com/FerretDB/FerretDB/issues/2733 // want "invalid TODO comment"
+func testIncorrectFormat() {
+	// TODO: https://github.com/FerretDB/FerretDB/issues/3413 // want "invalid TODO: incorrect format"
+}
+
+func testIncorrectClosed() {
+	// TODO https://github.com/FerretDB/FerretDB/issues/1 // want "invalid TODO: linked issue is closed"
```

Comment:
I updated tests to make error messages better. Please `git pull` and make them pass.

Reply:
@AlekSi how can i get the github token creds to run?

Reply:
They are not required. I can verify it yourself by running `curl https://api.github.com/repos/FerretDB/FerretDB/issues/2733` in the terminal without any credentials

Reply:
updated to pass test this should work now!! as per test message `//want `